### ISSUE TITLE
Remove pending requests from Playwright request queue after timeout

### ISSUE THREAD

**Issue by jirispilka** [apify/rag-web-browser#31]
Standby mode: request timeout - remove pending requests from Playwright request queue

**Comment by jirispilka** [comment]
It seems I’ll need to switch from `MemoryStorage` to `RequestQueue`:

```typescript
export declare abstract class RequestProvider implements IStorage {
```

Additionally, in standby mode, new requests shouldn’t be added to the forefront, as this might block earlier requests.

**Comment by MQ37** [comment]
@jirispilka is this issue still relevant? I tested with `requestTimeoutSecs=1` in standby and the request queue is empty. Only behaviour I noticed is it returns `{"errorMessage":"Timed out"}` but after a while the results are still added to the output dataset.

**Comment by jirispilka** [comment]
Yes, it's relevant—the response is sent without any results, but the crawler continues running for no reason. The user receives an empty response and likely won’t check the dataset.
